docs/src/basics.md

```diff
@@ -54,15 +54,15 @@ Most of the algorithms listed below also have a lazified version: see [Braun, Po
 
 ### Standard Frank-Wolfe (FW)
 
-It is implemented in the [`frank_wolfe`](@ref) function.
+It is implemented in the [`FrankWolfe.frank_wolfe`](@ref) function.
```

Comment:
all these shouldn't be needed I believe

Reply:
Okay, I just thought of adding that for consistency

Reply:
I would say it's not necessary, just adds clutter. If we can revert that then the rest is good